lapis2/src/main/kotlin/org/genspectrum/lapis/controller/LapisController.kt

```diff
@@ -584,10 +683,27 @@ private annotation class LapisAggregatedResponse
 @ApiResponse(
     responseCode = "200",
     description = "OK",
-    useReturnTypeSchema = true,
+    content = [
+        Content(schema = Schema(ref = "#/components/schemas/$NUCLEOTIDE_MUTATIONS_RESPONSE_SCHEMA")),
+    ],
 )
 private annotation class LapisNucleotideMutationsResponse
 
+@Target(AnnotationTarget.FUNCTION)
+@Retention(AnnotationRetention.RUNTIME)
+@Operation(
+    description = "Returns a list of mutations along with properties whose proportions are greater than or equal to " +
+        "the specified minProportion. Only sequences matching the specified sequence filters are considered.",
```

Comment:
"along with properties" - this sounds like the API would contain information on the properties of the mutations. Maybe write "counts and proportions" instead?

```suggestion
    description = "Returns a list of mutations along with the counts and proportions whose proportions are greater than or equal to " +
        "the specified minProportion. Only sequences matching the specified sequence filters are considered.",
```